Src/VimCore/EditorUtil.fs

```diff
@@ -2268,6 +2268,12 @@ module EditUtil =
                     inner (index + length)
         inner 0
 
+    /// Split a string into lines of text
+    let SplitLines (text: string) =
+        let text = RemoveEndingNewLine text
+        let text = NormalizeNewLines text "\n"

```

Comment:
Why `"\n"` here vs. using `EditUtil.NewLine` helper to get the preferred new line character?

Reply:
Because we are only temporarily normalizing the lines into something that `String.Split` can handle. The return value of the method is an array of strings without any newlines.

Reply:
Ah I see now.